### DISCUSSION TITLE
[CLI] stacked generate with line length format

### DISCUSSION THREAD

**Discussion by akifarhan** [FilledStacks/stacked#1109] [Q&A]
I am starting to use Stacked CLI. When I first `stacked create app` with a line-length option set to 120, the generated files followed the line length. But when I'm using the `stacked generate`, the generated files are formatted to 80. 

Based on my observation from the code base, it seems like at the end of the `stacked create` command, there's a `dart format` command to format the files to the desired line length while on the other hand, the `stacked generate` doesn't have it. 

Is it possible to run the `dart format` command when `stacked generate`? I want to create a PR for it but I'm not sure how to test the changes locally. Can someone guide me on it?

Thanks!

**Comment by FilledStacks** [comment]
Hey, yes that's possible. 

You think you can make a PR for that?

You can simply copy the code after create and move it to generate. I'll review and merge it in.

**Reply by akifarhan** [reply]
Ya, I can make a PR for that, but I'm not sure how to test it locally. 

I've tried this:
`dart pub global activate --source path .`

But received this error whenever I tried to run `stacked` commands. 
`FormatException: Could not parse ""/Users/akif/development/packages/stacked-org/cli"".`

Can you help me to get through this?

**Reply by FilledStacks** [reply]
I haven't seen that. 

Make the PR, I'll check if it works later and then push if it does.

**Reply by akifarhan** [reply]
Created a PR [here](https://github.com/Stacked-Org/cli/pull/46)
